### ISSUE TITLE
Validate image files are a reasonable size

### ISSUE THREAD

**Issue by kookster** [PRX/cms.prx.org#333]
should also validate on publish: https://github.com/PRX/publish.prx.org/issues/452

**Comment by svevang** [comment]
Some notes per my discussion with @cavis:

* There is broad symmetry here with the way the audio upload / lambda works. 
* Adding a new validation flag potentially changes flow control through the app.

We can generalize the approach taken so far in this PR by saving all the validation parameters that the lambda returns.  This has the effect of calculating validation errors in an "online" fashion, rather than caching them in a column.  Then we can tie set of error messages to form fields via the CMS api.

**Comment by cavis** [comment]
So I may have over-simplified this ticket, in the spirit of a coding challenge.  Before merging your PR, may be worth doing a design discussion about:

- these size restrictions are actually podcast/RSS specific, which is more a concern of Feeder not CMS
- we may want to resize images ourselves, either in the cms-image-lambda or Feeder
- or, if we say `image.purpose` indicates something about what sort of distribution the image is for, "profile" and "thumbnail" are pretty generic names

In other words ... validating in CMS solves the problem, but introduces some RSS-feed-esque business logic into the CMS app.
